server/plugin/command.go

```diff
@@ -339,7 +370,9 @@ func (p *Plugin) handleUnsubscribe(_ *plugin.Context, args *model.CommandArgs, p
 	}
 
 	repo := parameters[0]
-
+	if err := p.EnableNotificationTurnedOffRepo(repo); err != nil {
+		return "Encountered an error trying to remove from notify disabled list. Please try again."
```

Comment:
Why is `err` to getting wrapped?

Reply:
added a logWarn for the error if occurred

Reply:
I'm 1/5 that the code should handle errors in commands consistently which mean use wrapping. The error message might also be handy for the end user.

Reply:
should i remove this error check. 
we have already an previous error handling for unsubscribe part .